app/models/aliquot.rb

```diff
@@ -9,6 +9,8 @@ class Aliquot < ActiveRecord::Base
   include Api::Messages::FlowcellIO::AliquotExtensions
   include AliquotIndexer::AliquotScopes
 
+  TAG_COUNT_NAMES = ['Untagged','Single','Dual']
+
```

Comment:
Tag number was confusing, as it was unclear if it applied to per sample, or per pool. I've opted for names instead, to make it more apparent.